src/widgetresize.js

```diff
@@ -95,15 +112,58 @@ export default class WidgetResize extends Plugin {
 
 		resizer.attach();
 
-		this.editor.editing.view.once( 'render', () => resizer.redraw() );
-
-		this.resizers.push( resizer );
+		this.resizers.set( options.viewElement, resizer );
 
 		return resizer;
 	}
 
+	/**
+	 * Adds a listener that keep the track of currently focused resizer.
```

Comment:
This method's description doesn't match what it really does. It doesn't track focus. It redraws the focused resizer. 

It violates SRP and that's one of the reasons why I'd improve it.

I also don't like the fact that we do redraw() here and in the ui#update listener above. It's good to have those things kept together. To have better clarity when redraws happen.

For instance, one option would be to have `focusedResizer` observable and in `init()` register a `change:focusedResizer` listener and redraw call there.

Finally, I'd be ok with having this at least protected, if not public. May be quite helpful to write tests for this. And `visibleResizer` would be better, as the resizer itself is never focused. 

BTW, if you'll be making `visibleResizer` observable, it'd be good to make `activeResizer` observable too for consistency.

Reply:
It was removed due to https://github.com/ckeditor/ckeditor5-widget/pull/108#discussion_r324335108